packages/cxl-institute/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@conversionxl/cxl-institute",
+  "version": "0.1.0",
+  "author": "CXL <leho@conversionxl.com>",
+  "license": "SEE LICENSE IN LICENSE.md",
+  "main": "src/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/conversionxl/aybolit",
+    "directory": "packages/cxl-institute"
+  },
+  "dependencies": {
+    "@conversionxl/cxl-lumo-styles": "^0.1.0",
+    "@conversionxl/cxl-ui": "^0.1.0",
+    "@pika/pack": "^0.3.0",
+    "@pika/plugin-build-web": "^0.3.0",
+    "@pika/plugin-bundle-web": "^0.3.0",
+    "@pika/plugin-standard-pkg": "^0.3.0",
+    "@polymer/iron-icon": "^3.0.1",
+    "@vaadin/vaadin-button": "^2.1.5",
+    "@vaadin/vaadin-details": "^1.0.1",
+    "lit-element": "^2.1.0",
+    "sass-mq": "^5.0.0"
```

Comment:
@chanar is `sass-mq` actually a devDependency, or? How is it involved in post-compile runtime?

Reply:
Yes correct, this should be devDep. Same thing with cxl-ui package.